src/Definition/Exasol.php

```diff
@@ -0,0 +1,406 @@
+<?php
+
+namespace Keboola\Datatype\Definition;
+
+use Keboola\Datatype\Definition\Exception\InvalidLengthException;
+use Keboola\Datatype\Definition\Exception\InvalidOptionException;
+use Keboola\Datatype\Definition\Exception\InvalidTypeException;
+
+/**
+ * Class Exasol
+ *
+ * https://docs.exasol.com/sql_references/data_types/datatypedetails.htm
+ */
+class Exasol extends Common
+{
+    /* core types */
+    const TYPE_DECIMAL = 'DECIMAL'; // DECIMAL(p,s) = s ≤ p ≤ 36
+    const TYPE_DOUBLE_PRECISION = 'DOUBLE PRECISION';
+    const TYPE_BOOLEAN = 'BOOLEAN';
+    const TYPE_DATE = 'DATE';
+    const TYPE_TIMESTAMP = 'TIMESTAMP';
+    const TYPE_TIMESTAMP_WITH_LOCAL_ZONE = 'TIMESTAMP WITH LOCAL TIME ZONE';
+    const TYPE_INTERVAL_YEAR_TO_MONTH = 'INTERVAL_YEAR TO MONTH'; // INTERVAL_YEAR [(p)] TO MONTH
```

Comment:
@jirkasemmler 
Podle ostatních hodnot tady je asi typo. Nemá to být `INTERVAL YEAR TO MONTH` (bez podtržítka)?

Reply:
jj, je to pak opravene v https://github.com/keboola/php-datatypes/blob/master/src/Definition/Exasol.php#L23

Reply:
Díky, to jsem si nevšiml.